clients/tabby-agent/src/status.ts

```diff
@@ -205,6 +208,8 @@ export class StatusProvider extends EventEmitter implements Feature {
           const ignored = this.dataStore.data.statusIgnoredIssues ?? [];
           if (this.tabbyApiClient.hasCompletionResponseTimeIssue() && !ignored.includes("completionResponseSlow")) {
             statusInfo = { status: "completionResponseSlow" };
+          } else if (this.tabbyApiClient.isRateLimited()) {
+            statusInfo = { status: "rateLimited" };
           } else if (this.tabbyApiClient.isFetchingCompletion()) {
```

Comment:
higher priority than  `completionResponseSlow`

```suggestion
          if (this.tabbyApiClient.isRateLimited()) {
            statusInfo = { status: "rateLimited" };
          } else if (this.tabbyApiClient.hasCompletionResponseTimeIssue() && !ignored.includes("completionResponseSlow")) {
            statusInfo = { status: "completionResponseSlow" };
          } else  if (this.tabbyApiClient.isFetchingCompletion()) {
```